src/coreclr/src/jit/emitarm.cpp

```diff
@@ -3513,8 +3513,10 @@ void emitter::emitIns_S(instruction ins, emitAttr attr, int varx, int offs)
 /*****************************************************************************
  *
  *  Add an instruction referencing a register and a stack-based local variable.
+ *
+ *  Stores the base register used for accessing the offset of stack in "pBaseReg".
  */
-void emitter::emitIns_R_S(instruction ins, emitAttr attr, regNumber reg1, int varx, int offs, regNumber* baseReg)
+void emitter::emitIns_R_S(instruction ins, emitAttr attr, regNumber reg1, int varx, int offs, regNumber* pBaseReg)
```

Comment:
It would be nice to change this to a standard function header: https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/clr-jit-coding-conventions.md#94-function-header-comment (we have been gradually adding these as we touch the many methods that don't yet have them), but I wouldn't necessarily hold this up for that.